Benchmark/arithmetic.jl

```diff
@@ -0,0 +1,40 @@
+# ==========
+# Arithmetic
+# ==========
+
+SUITE["Arithmetic"] = BenchmarkGroup()
```

Comment:
Perhaps make two separate "suite", `SUITE["Arithmetic - scalar"]` and `SUITE["Arithmetic - vector"]` ?